pkg/cinderapi/const.go

```diff
@@ -1,8 +1,21 @@
+/*
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
 package cinderapi
 
 const (
-	// AppLabel -
-	AppLabel = "cinder-api"
 	// KollaConfig -
 	KollaConfig = "/var/lib/config-data/merged/cinder-api-config.json"
```

Comment:
you deleted that template, , I guess we need one

Reply:
I'm not sure I understand what you mean here.  There are `cinder-*-config.json` templates that get rendered in the config map associated with the Cinder CR (the umbrella CR for all the other services).  For example, for the API service: https://github.com/abays/cinder-operator/blob/new_structure/templates/cinder/config/cinder-api-config.json.  The other sub-service CRs then mount the config map that include these `*.json` files.  They are able to use these in their respective pods  to ensure that config files are copied into place for use with the actual service containers.

Reply:
ah ok, sorry for that I haven't checked the repo. for some reason I expected them here in the review.